CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## v2.2.0
+
+* Added the input `/credit_card/country`. This is the country where the
```

Comment:
The tense on this is different than the other entries! I don't have a strong opinion on which is "right" as long as the document is consistent.

Reply:
Yeah, I was thinking we could switch actually as the other way is different from the other APIs. It'd be simpler if we were more consistent across the APIs. That said I'm ok changing it too.

Reply:
Ah, ok. That seems fine.